dashboard-app/src/components/UnitToggle.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Toggle from './Toggle';
+
+
+type DurationUnit = 'Hours' | 'Days';
```

Comment:
I've defined this as a `UnitEnum` on my current branch. Wanna do the same here, then it can be refactored in mine. I can't think of any benefits of it being a union type, and an enum will allow for it to be easily passed into copy.